src/main/java/life/qbic/portal/portlet/Project.java

```diff
@@ -0,0 +1,36 @@
+package life.qbic.portal.portlet;
+
+/**
+ * <short description>
+ * <p>
+ * <detailed description>
+ *
+ */
```

Comment:
Again, a description of the class would be awesome :)

Reply:
9fd8aa1